core/src/ldml/ldml_transforms.cpp

```diff
@@ -175,13 +438,28 @@ transforms::apply(std::u16string & str) {
   return true;
 }
 
+bool
+transforms::apply(std::u32string &str) {
+  bool rc = false;
+  // TODO-LDML: PoC implementation for now, need to refactor into fcns
+  // ONLY reorder
+  for (auto group = transform_groups.begin(); group < transform_groups.end(); group++) {
+    assert(group->type == reorder);  // TODO-LDML
+    auto rgroup = group->reorder;
+    if (rgroup.apply(str)) {
+      rc = true;
```

Comment:
Any reason not to just `return true` here and shortcircuit?

Reply:
yes! that would only apply the _first_ transform group.  We need to apply all of them.

Reply:
duh, yes of course ... it's a "did we apply anything" result, not a "did we succeed" result...